internal/cmd/operator_list_custom_resources.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	"github.com/operator-framework/kubectl-operator/internal/pkg/action"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func newOperatorListCustomResourcesCmd(cfg *action.Configuration) *cobra.Command {
+	CRLister := action.NewOperatorListCustomResources(cfg)
+	cmd := &cobra.Command{
+		Use:     "list-custom-resources",
+		Aliases: []string{"list-crs"},
+		Short:   "List all custom resources on-cluster associated with a particular operator",
+		Args:    cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
+				log.Fatal("did not provide operator or package name")
+			}
+			CRLister.PackageName = args[0]
+
+			op, err := CRLister.FindOperator(cmd.Context())
+			if err != nil {
+				log.Fatal(err)
+			}
+			fmt.Printf("found operator %s", op.Name)
```

Comment:
`kubectl` doesn't write log messages like this unless verbosity is set very high. Since `kubectl operator` doesn't have verbosity (at least not yet), I recommend removing these print statements.

Reply:
switching to using the `log` library like in other commands